Scripts/manage-version.sh

```diff
@@ -22,7 +22,7 @@ OUTPUT_GREEN="\033[32m"
 OUTPUT_BOLD="\033[1m"
 
 # Config files
-publicConfig=("Version.public.xcconfig") 
+publicConfig=("Version.public.xcconfig")
```

Comment:
I have a feeling this file, too, ought to be removed. I can't find a usage for it either [locally](https://github.com/wordpress-mobile/WordPress-iOS/search?q=manage-version) or in the [release toolkit](https://github.com/wordpress-mobile/release-toolkit/search?q=manage-version)

🤔

Reply:
+1